### ISSUE TITLE
Additional option to make the Forms become an overlay without rerouting

### ISSUE THREAD

**Issue by andmitsch** [meteor-useraccounts/core#434]
It would be nice to have an option that allows login forms to be shown in a dropdown or popup or a overlay without a reroute happening. However, with having a fixed route accessible for fallback reasons, e.g. if aforementioned option is not a choice like on few mobiles.

**Comment by splendido** [comment]
...for the dropdown stuff theres `accounts-ui` ;-)
something like a modal?

This might me something for v2.0

**Comment by andmitsch** [comment]
Yes, like a modal, but also other kind of overlays or dropdowns.

Thanks though ^^
